quic/s2n-quic-tests/tests/testsuite/main.rs

```diff
@@ -1,7 +1,7 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Why not just continue to put the tests in `src/tests.rs`? IIRC it should be faster to compile since you don't have to compile and link the intermediate `s2n-quic-tests` lib.

Reply:
Right. That post mentions exactly what I'm talking about:

> For an internal library, I avoid integration tests all together. Instead, I use Cargo unit tests for “integration” bits:
> 
> ```
> src/
>  lib.rs
>  tests.rs
>  tests/
>    foo.rs
>    bar.rs
> ```
>
> That way, I avoid linking the separate integration tests binary altogether. I also have access to non-pub API of the crate, which is often useful.